### PR TITLE
WIE 8/our team section

### PR DESCRIPTION
**Pull request by RosaPak**
**Completed 'About Us' - Our Team Section.**

- Added Title + Description
- Implemented Team pictures!
- Mobile responsiveness added

image

### CODE REVIEW

File: app/(pages)/about-us/page.tsx [hunk 1]
```diff
@@ -41,6 +44,49 @@ function AboutUsCard({image, cardName, cardText, link}: cardProps) {
...
+            </AspectRatio>
+          </Container> 
+          <Container 
+            sx={{
+              width:'100%',
+              display: 'flex',
+              flexDirection: 'column',
+              alignItems: 'center',
+              paddingTop: '5%',
+              paddingBottom: '20%'
```
Comment (angewen) [inline app/(pages)/about-us/page.tsx:74]: Are these the same as pt and pb?

File: app/(pages)/about-us/page.tsx [hunk 2]
```diff
@@ -41,6 +44,49 @@ function AboutUsCard({image, cardName, cardText, link}: cardProps) {
...
+            }}>
+            <Typography level='subtitle' className='graphite' textAlign='center' pt={3}>
+              {teamRoles[index]}
+            </Typography>
+            <Typography level='subtitle-light' className='graphite' textAlign='center'>
```
Comment (angewen) [inline app/(pages)/about-us/page.tsx:79]: these fonts already default to graphite, see ThemeRegistry.tsx

**Review by angewen — CHANGES_REQUESTED** [review]

File: app/(pages)/about-us/page.tsx [hunk 3]
```diff
@@ -97,6 +143,17 @@ export default function AboutUs() {
...
+      <Sheet>
+        <Typography level='h1' className='graphite' textAlign='center' py={2} sx={{marginBottom: 0}}>
+          Our Team
+        </Typography>
+        <Typography level='body' className='graphite' textAlign='center' pb={7}>
+          Meet the team behind the magic!
+        </Typography>
+        <OurTeamProfiles>
+
+        </OurTeamProfiles>
```
Comment (sophiac6410) [inline app/(pages)/about-us/page.tsx:155]: You can use one tag <OutTeamProfiles /> instead

File: app/(pages)/about-us/page.tsx (hunk 1 above)
Comment (RosaPak) [reply app/(pages)/about-us/page.tsx:74]: Yep, I can change them to be pt, pb

File: app/(pages)/about-us/page.tsx (hunk 2 above)
Comment (RosaPak) [reply app/(pages)/about-us/page.tsx:79]: fixed!

File: app/(pages)/about-us/page.tsx (hunk 3 above)
Comment (RosaPak) [reply app/(pages)/about-us/page.tsx:155]: fixed!

**Review by angewen — APPROVED** [review]
LGTM

**Pull request merged**